Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -1810,18 +1810,25 @@ def test_get_closure_case(params, expected):
     assert get_closure_case(params) == expected
 
 
-@pytest.mark.parametrize('ticket_state, expected_res', [('1', 'Other'),
-                                                        ('7', 'Resolved')])
-def test_converts_state_close_reason(ticket_state, expected_res):
+@pytest.mark.parametrize('ticket_state, server_close_custom_state, expected_res',
+                         [('1', None, 'Other'),
+                          ('7', None, 'Resolved'),
+                          ('6', None, 'Resolved'),
+                          ('10', '10=Test', 'Test'),
+                          ('10', '10=Test,11=Test2', 'Test'),
+                          (None, None, 'Other'),
```

Comment:
Lets fix this test to make sure empty custom states == Not closing.

Reply:
Done.